docs/service-brokers/docs/020-architecture-reb.md

```diff
@@ -3,25 +3,17 @@ title: The Remote Environment Broker architecture
 type: Architecture
 ---
 
-The Remote Environment Broker (REB) workflow starts with the registration process, during which a remote environment (RE) is registered on the Kyma cluster.
+The Remote Environment Broker (REB) workflow consists of the following steps:
 
-### Remote environment registration process
+1. The Remote Environment Broker watches for RemoteEnvironments (REs) in the cluster and EnvironmentMappings (EMs) in all Namespaces.
```

Comment:
and not all Environments?

Reply:
Actually, originally I've also written `all Environments` but during the technical review, I was asked to change Environments into Namespaces.

Reply:
@polskikiel Could you explain why we should use Namespace instead of Environment here?

Reply:
@klaudiagrz @kazydek From the technical point of view REB watches for EnvironmentMappings in all Namespaces. Not only in the Environments.